blueocean-pipeline-editor/src/main/js/components/editor/InputTextArea.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+function cn(...args) {
+    let out = null;
+    for (let i = 0; i < args.length; i++) {
```

Comment:
Alternatively I think you could simplify this method down to `args.filter(arg => !!arg).join(' ')` ?